packages/@ama-sdk/core/package.json

```diff
@@ -42,6 +42,7 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
+    "@angular-devkit/schematics": "~15.1.3",
```

Comment:
Should be at least optional.
The TS SDKs should not require the end user to bring any `@angular` deps :S

Reply:
yes, it's used only here in the schematic script, let;s turn it optional